src/main/java/org/dice_research/opal/catfish/Catfish.java

```diff
@@ -1,25 +1,68 @@
 package org.dice_research.opal.catfish;
 
-import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.*;
+import org.apache.jena.vocabulary.DCAT;
+import org.apache.jena.vocabulary.SKOS;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.dice_research.opal.common.interfaces.JenaModelProcessor;
 import org.dice_research.opal.common.interfaces.ModelProcessor;
 
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+
 @SuppressWarnings("deprecation")
 public class Catfish implements ModelProcessor, JenaModelProcessor {
 
 	private static final Logger LOGGER = LogManager.getLogger();
 
+	ArrayList<Statement> deletingStatements = new ArrayList<Statement>();
+
 	@Override
 	public void processModel(Model model, String datasetUri) throws Exception {
 
 		// TODO: clean model
 
+		Resource dataset = ResourceFactory.createResource(datasetUri);
```

Comment:
Create a new class for cleaning themes. That class should have a method like `processModel(Model model, String datasetUri)`